app/views/sections/show.html.slim

```diff
@@ -46,32 +53,35 @@
               .category-title
                 h4= category.name
                 p= category.description
-              - unless category.videos.empty?
+              - if category.highlight.presence
```

Comment:
`presence` returns the value if present. `present?` checks if a value is present. Use `present?` in this case.